src/server/worker/driver/driver.go

```diff
@@ -41,7 +42,7 @@ import (
 // TaskNamespace returns the namespace used by the task package for this
 // pipeline.
 func TaskNamespace(pipelineInfo *pps.PipelineInfo) string {
-	return fmt.Sprintf("/pipeline-%s", ppsdb.VersionKey(pipelineInfo.Pipeline, pipelineInfo.Version))
+	return fmt.Sprintf("/pipeline-%s", url.QueryEscape(ppsdb.VersionKey(pipelineInfo.Pipeline, pipelineInfo.Version)))
```

Comment:
is it better to do something like
```suggestion
	return url.QueryEscape(fmt.Sprintf("/pipeline-%s", ppsdb.VersionKey(pipelineInfo.Pipeline, pipelineInfo.Version)))
```

Reply:
I don’t believe so, because I think that first `"/"` needs to be passed through unchanged to etcd.